### PR TITLE
Add more builtin images: smilies, geometry, musical notes, and more.

### PR DESCRIPTION
**Pull request by dpgeorge**
This patch adds all the images from @marichkakorolyuk.

They are really awesome!

Would be good to get feedback on the names of the images.  Some (eg musical_note_beamed) could probably do with better names.

### CODE REVIEW

File: source/microbit/microbitconstimage.cpp [hunk 1]
```diff
@@ -284,4 +316,132 @@ IMAGE_T microbit_const_image_arrow_nw_obj = SMALL_IMAGE(
...
+
+// musical notes
+
+IMAGE_T microbit_const_image_musical_note_obj = SMALL_IMAGE(
+    0,0,1,0,0,
+    0,0,1,1,0,
+    0,0,1,0,1,
+    1,1,1,0,0,
+    1,1,1,0,0
+);
```
Comment (ntoll) [inline source/microbit/microbitconstimage.cpp]: music_crotchet?

File: source/microbit/microbitconstimage.cpp [hunk 2]
```diff
@@ -284,4 +316,132 @@ IMAGE_T microbit_const_image_arrow_nw_obj = SMALL_IMAGE(
...
+
+IMAGE_T microbit_const_image_musical_note_beamed_obj = SMALL_IMAGE(
+    0,1,1,1,1,
+    0,1,0,0,1,
+    0,1,0,0,1,
+    1,1,0,1,1,
+    1,1,0,1,1
+);
```
Comment (ntoll) [inline source/microbit/microbitconstimage.cpp]: music_quavers?

**Comment by ntoll** [comment]
I'd like to add a rocket too:

```
..*..
.***.
.***.
.***.
**.**
```

**Comment by dpgeorge** [comment]
> I'd like to add a rocket too:

TTP just went to zero!

**Comment by ntoll** [comment]
Dammit... :-)

**Comment by markshannon** [comment]
:+1: :rocket:

**Comment by dpgeorge** [comment]
Changed the names of the musical images to that suggested by @ntoll and merged in cc430180c412fb9d54b68f83b5e15ca96d03fcf7.  I did not add the rocket... feel free to do it!

**Pull request closed without merging**